src/Numeric/Units/Dimensional/Internal.hs

```diff
@@ -137,6 +140,10 @@ instance (Num a) => Monoid (SQuantity s d a) where
   mempty = Quantity 0
   mappend = (<>)
 
+#if USE_BINARY
+deriving instance (Data.Binary.Binary a) => Data.Binary.Binary (SQuantity s d a)
```

Comment:
@dmcclean had put the instance on `Quantity d a`. Since `Quantity` is a specialization of `SQuantity Data.ExactPi.TypeLevel.One`, I guess it is OK and more generic to derive for `SQuantity`? And there is no need to derive instances for the phantom types? Probably stupid questions but I never used or investigated binary so please confirm. 😄

Reply:
At the time, [`SQuantity` didn't exist!](https://github.com/bjornbm/dimensional/blob/c5d84fc151616d410983f08a12b2c34901fa408c/src/Numeric/Units/Dimensional/Internal.hs)

`SQuantity` is also a type synonym -- unfortunately I'm not a type wizard (yet anyway), so I'm not sure how to make `Dimensional` an instance of `Binary`. So I closely followed what @dmcclean did